File: vue/src/api/es-map.js
import request from '@/utils/request'

const api = '/api/es_map/'

export function ListAction(data) {
  return request({
    url: api + 'ListAction',
    method: 'post',
    data
  })
}
export function UpdateMappingAction(data) {
  return request({
    url: api + 'UpdateMappingAction',
    method: 'post',
    data
  })
}

File: pkg/web/es_map.go
package web

// ES mapping 路由
func (this *WebServer) runEsMap() {

	const AbsolutePath = "/api/es_map"
	group := this.engine.Group("ES映射", "/api/es_map")
	{
		group.POST(false, "查看mapping列表", "/ListAction", this.esMappingController.ListAction)
		group.POST(false, "修改mapping", "/UpdateMappingAction", this.esMappingController.UpdateMappingAction)

	}
}
